src/main/java/com/csv/communitytrackerjava/service/ProjectServiceImpl.java

```diff
@@ -77,16 +79,29 @@ public ProjectResponseDTO updateProject(ProjectUpdateDTO projectUpdateDTO, int i
     @Override
     public ProjectResponseDTO findAllProject() {
         ProjectPayloadDTO payloadDTO = new ProjectPayloadDTO();
-        payloadDTO.setAdditionalProperty("projects", projectMapper.toListDTO(projectRepository.findAll()));
+        payloadDTO.setAdditionalProperty("projects", projectRepository.findAll());
         return toProjectResponseDTO("Successfully fetch all projects.", payloadDTO);
     }
-    
-    public ProjectResponseDTO toProjectResponseDTO(String message, ProjectPayloadDTO payloadDTO){
+
+    @Override
+    public Page<ProjectGetPeopleDTO> findPeopleByProjectId(Pageable pageable, Set<Integer> id) throws Exception {
+        List<ProjectGetPeopleDTO> projectList = (
+                projectRepository.findAllByProjectIdIn(id, pageable)
+                        .stream()
+                        .map(projects -> projectMapper.toGetPeopleDTO(projects))
```

Comment:
You can use lambda expression here with declaring projects variables as you're passing the project entity right ahead.

sample. projectMapper::toGetPeopeleDto